grammar.js

```diff
@@ -483,6 +483,8 @@ module.exports = grammar({
     // :h lambda
     lambda_expression: ($) =>
       seq('{', commaSep($.identifier), '->', $._expression, '}'),
+
+    _escaped_line: ($) => alias($._line_continuation, '\\'),
```

Comment:
I don't see a place where this is used, is it normal ?
Maybe you reather want to change this in the `extras` key too ?

Reply:
I don't know if it's normal, but it does help to show the tokens. Including `_escaped_line` in `extras` should be the correct way, but it leads to a long wait time when generating the grammar... we may want to ask the tree-sitter devs about this situation, may be a bug (updating the tree-sitter-cli doesn't help)

Reply:
No but what I mean is remove every instance of `_line_continuation` and replace it by `_escaped_line` in here

Reply:
Appart from the on in `externals` and in this rule definition

Reply:
`_line_continuation` is only used on `externals`